ckanext/spatial/lib/csw_client.py

```diff
@@ -66,6 +66,10 @@ class CswService(OwsService):
     Perform various operations on a CSW service
     """
     from owslib.csw import CatalogueServiceWeb as _Implementation
+
+    def __init__(self):
+        self.sortby = SortBy([SortProperty('dc:identifier')])
```

Comment:
We need a `super(CswService, self).__init__()` here, otherwise we'll lose whatever happens on the parent class `__init__` method.